Source/Core/DolphinQt/ConvertDialog.cpp

```diff
@@ -156,21 +148,17 @@ void ConvertDialog::OnFormatChanged()
   {
   case DiscIO::BlobType::GCZ:
   {
-    // In order for versions of Dolphin prior to 5.0-11893 to be able to convert a GCZ file
-    // to ISO without messing up the final part of the file in some way, the file size
-    // must be an integer multiple of the block size (fixed in 3aa463c) and must not be
-    // an integer multiple of the block size multiplied by 32 (fixed in 26b21e3).
-
+    // To support legacy versions of dolphin, we have to check the GCZ block size
+    // See DiscIO::IsGCZBlockSizeLegacyCompatible() in DiscUtil.cpp for details
```

Comment:
`DiscUtils.cpp`. (Alternatively, if you prefer it, don't mention the filename. `IsGCZBlockSizeLegacyCompatible` is a unique enough function name that it's easy to search for.)